flycheck.el

```diff
@@ -7518,6 +7513,14 @@ See URL `https://github.com/eslint/eslint'."
                                    (flycheck-error-message err))))
                           (flycheck-sanitize-errors errors))
                   errors)
+  :enabled (lambda (checker)
+             (let* ((directory (concat
+                                (flycheck-compute-working-directory checker)
+                                "."))
```

Comment:
In my original implementation I used:
`(tempfile (flycheck-save-buffer-to-temp #'flycheck-temp-file-inplace))`, however using `flycheck-compute-working-directory` should be faster.